pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -186,3 +186,12 @@ def get_download_url(self, obj):
         filename_path = self.context["content_artifact"].relative_path.lstrip("/")
         download_url = f"{host}/{distro_base_path}/{filename_path}"
         return download_url
+
+
+class CollectionVersionDocsSerializer(CollectionVersionSerializer):
```

Comment:
Can we split this from the `CollectionVersionSerializer` also? Maybe it would just be:

```
class CollectionVersionDocsSerializer(serializers.ModelSerializer):
    """A serializer to display the docs_blob of a CollectionVersion."""

    docs_blob = serializers.JSONField()

    class Meta:
        fields = ("docs_blob",)
        model = models.CollectionVersion
```